greenwave/subjects/subject.py

```diff
@@ -106,6 +117,10 @@ def result_queries(self):
         else:
             yield self.to_dict()
 
+    def _matching_product_versions(self, text, match_dict):
```

Comment:
match_dict is not used at all.

Reply:
Ha, not sure why tests did not catch this.

Reply:
Should be fixed now and properly verified in tests.